libs/reglibs.go

```diff
@@ -0,0 +1,69 @@
+package libs
+
+import (
+	"embed"
+	"fmt"
+	"io/fs"
+	"strings"
+)
+
+func init() {
+	RegisterLibraries()
+}
+
+//go:embed includes/*
+var efs embed.FS
+
+func RegisterLibraries() {
+
+	for {
+		fNames, err := fileNames()
+
+		if err != nil {
+			break
```

Comment:
```suggestion
			panic(err)
```
same for `efs.ReadFile`